testutils/logger.go

```diff
@@ -0,0 +1,12 @@
+package testutils
+
+import (
+	"log/slog"
+	"os"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+)
+
+func GetTestLogger() logging.Logger {
+	return logging.NewTextSLogger(os.Stdout, &logging.SLoggerOptions{Level: slog.LevelDebug})
```

Comment:
can we add source info as well?

Reply:
7938384f45812db3b835c78090aeb94b6c1ca7fe